server/src/platform/drbg/helpers.rs

```diff
@@ -0,0 +1,52 @@
+use super::Drbg;
+
+/// Implement `fill_bytes` via `next_u64` and `next_u32`, little-endian order.
+///
+/// The fastest way to fill a slice is usually to work as long as possible with
+/// integers. That is why this method mostly uses `next_u64`, and only when
+/// there are 4 or less bytes remaining at the end of the slice it uses
+/// `next_u32` once.
+pub fn fill_bytes_via_next<R: Drbg>(
+    rng: &mut R,
+    additional_input: &[u8],
+    dest: &mut [u8],
+) -> Result<(), R::Error> {
+    let mut left = dest;
+    while left.len() >= 8 {
+        let (l, r) = { left }.split_at_mut(8);
+        left = r;
+        let chunk: [u8; 8] = rng.next_u64(additional_input)?.to_le_bytes();
```

Comment:
Do we *really* need to force this into little-endian bytes specifically? It's pseudo-random bits, what's the difference between us grabbing bits and just casting to a slice without an endian conversion versus not (`to_ne_bytes`)?

Reply:
I didn't put effort into that, they do the same in rust's own rng library. But the idea is we want the algorithm to produce same results regardless of which platform it is running on. (one reason would be testing)